fs/xfstests.py.data/btrfs-subpage.yaml

```diff
@@ -2,7 +2,8 @@ skip_dangerous: True
 scratch_mnt: '/mnt/scratch'
 test_mnt: '/mnt/test'
 disk_mnt: '/mnt/loop-device'
-test_range: '54-55'
+group: 'auto'
+test_range: null
```

Comment:
for all below yamls.. if there is no value.. keep it empty.  thats the standard we follow .. no need to provide null
in code when params.get  keep default value empty

Reply:
updated all the yaml files